src/coreclr/scripts/superpmi.py

```diff
@@ -147,6 +147,7 @@
 spmi_location_help = """\
 Directory in which to put SuperPMI files, such as downloaded MCH files, asm diffs, and repro .MC files.
 Optional. Default is 'spmi' within the repo 'artifacts' directory.
+If 'SPMI_LOCATION' environment variable is set to a path, it will use that directory.
```

Comment:
I think this is fine, and matches the argument. But I wonder if we should be more verbose and consistent for environment variables, and use `SUPERPMI_LOCATION`.

I never liked the "location" name, but I don't have a better idea. `SUPERPMI_CACHE_DIRECTORY`?

Reply:
sure, may be `SUPERPMI_CACHE_DIR` to keep it short?

Reply:
actually, i would just keep it `SUPERPMI_CACHE_DIRECTORY`. Sounds better and clear.